server/text_generation_server/utils/weights.py

```diff
@@ -9,6 +10,15 @@
 from text_generation_server.utils.log import log_once
 
 
+@dataclass
+class _GPTQParams:
```

Comment:
not important: but is there a reason this class is prefixed with `_`?

Reply:
To make it extra clear that the class is not meant for external consumption (not a public API).